tests/_support/FunctionalTester.php

```diff
@@ -69,6 +70,56 @@ public function grabHtmlContentFromLastSentEmail(): ?string
         return $this->grabHtmlContentFromEmail($email->getSwiftMessage());
     }
 
+    public function haveAdminSurvey(): Survey
+    {
+        if (!isset($this->adminSurvey)) {
+            $this->adminSurvey = $survey = new Survey();
+            $survey->config = [
+                'pages' => [
+                    0 => [
```

Comment:
This index is not needed (I suspect you generated this PHP with a JSON -> PHP converter)